### PR TITLE
WIP-524- FE updated to reflect the changes made in API regarding the …

### PR DESCRIPTION
**Pull request by superspartak**
…features values being injected to assignments in subproject data

### CODE REVIEW

File: src/components/templates/SubOrderSection/SubOrderSection.tsx [hunk 1]
```diff
@@ -133,20 +125,18 @@ const SubOrderSection: FC<SubOrderProps> = ({
...
-      correspondingAssignments,
-      ({ assigned_vendor_id }) => !assigned_vendor_id
-    )
-  })
+  const hasAnyUnassignedFeatures = !find(
+    assignments,
+    ({ assignee }) => !assignee
+  )
+
+  const tabs = assignments.map(
```
Comment (IngaSokman) [inline src/components/templates/SubOrderSection/SubOrderSection.tsx]: For consistency reasons would be nice to use already imported map from lodash

**Review by IngaSokman — APPROVED** [review]

**Comment by VaabelMaret** [comment]
[blocker]
catSupported now comes from a different place. `project.type_classifier_value.project_type_config.cat_tool_enabled`

File: src/components/molecules/Assignment/Assignment.tsx [hunk 2]
```diff
@@ -79,6 +79,8 @@ const Assignment: FC<AssignmentProps> = ({
     [vendor]
   ) as DiscountPercentages
 
+  const feature = job_definition.job_key
```
Comment (VaabelMaret) [inline src/components/molecules/Assignment/Assignment.tsx:82]: If there isn't any job_key,  this part might give as an error

**Review by VaabelMaret — CHANGES_REQUESTED** [review]

File: src/components/molecules/Assignment/Assignment.tsx (hunk 2 above)
Comment (superspartak) [reply src/components/molecules/Assignment/Assignment.tsx:82]: Dima confirmed that it can never be null

**Review by VaabelMaret — APPROVED** [review]

**Pull request merged**